tests/slurm-tests/slurm_test_llama_nemotron_super_49B_v1._5_evals/check_eval_results.py

```diff
@@ -0,0 +1,258 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+import io
+import json
+import os
+
+REASONING_TASKS = [
+    "math-500",
+    "aime24",
+    "aime25",
+    "gpqa",
+    "mmlu-pro",
+    "livecodebench",
+    "scicode",
+    "hle",
+]
+
+REASONING_BENCHMARKS_SCIENCE_HLE = {"scicode", "hle"}
+
+REASONING_REQUIRED_FIELDS = {
+    "math-500": ["symbolic_correct"],
+    "aime24": ["symbolic_correct"],
+    "aime25": ["symbolic_correct"],
+    "gpqa": ["symbolic_correct"],
+    "mmlu-pro": ["symbolic_correct"],
+    "livecodebench": ["accuracy"],
+    "scicode": ["problem_accuracy", "subtask_accuracy"],
+    "hle": ["judge_correct", "symbolic_correct"],
+}
+
+REASONING_METRIC_RANGES = {
+    "reasoning_on": {
```

Comment:
can you narrow down some of these please? We can expand later if needed. I suggest the following

For aime, let's use 10 points as you currently have. For math-500, let's use 3 points around mean. For gpqa 5 points. For mmlu-pro 3 points. For lcb 5 points. For scicode, maybe also 5 points, but please update the numbers, we just changed them in the tutorial. For hle, no need to check symbolic correct and let's have 3 points around mean for judge.